sponsor_us.md

```diff
@@ -17,40 +17,46 @@ By supporting us you help to make the world a little better. Your company also
 increase diversity in the technology scene.
 
-Sponsors are listed on our website and on our event page on ClojureBridge.org.
-They receive a shoutout at the start and end of the event, and receive a warm
-"thank you" on our social media account. We also list sponsors by name on our
-popular t-shirts that are handed out to teachers and attendees, and provide the
-opportunity to offer "swag" (stickers, note books, thumb drives, etc.) at the
-event.
+Your company has an option of different tiers of support. There's the direct
+sponsorship tiers, which are the most valuable form of support, and a generic
+small supporter tier. The direct sponsorship tiers involve directly paying our
+supplier with the order details we provide, and as such also mean you can pay
+with credit card will and receive an invoice.
 
-## Financial
+* Platinum tier (~600€): Food for the main event 
+* Gold Tier (~500€): Notebooks for the Attendees 
+* Silver Tier (~400€): Food for the coaches training
+* Small Supporter Tier (250€)
 
-To handle donations we work with
-[Travis Foundation UG](http://foundation.travis-ci.org/), a registered
-not-for-profit company (a “Gemeinnützige Organisation” under German law.) Travis
-Foundation can issue a Donation Receipt for accounting and tax deduction. It
-does *not* issue invoices for donations.
+All our sponsors are listed on our website and on our event page on 
+ClojureBridge.org. They'll receive a shoutout at the start and end of the event
+along with their logos in our presentations. All our sponsors will also have
+the opportunity to offer "swag" (stickers, note books, thumb drives, etc.) at 
+the event.
+
+Additionally, those who sponsored in our top three tiers will also get special
+shoutouts on our social media accounts, along with priority placements of their
+logos.
 
-As a sponsor you can also choose to directly pick up the bill for one of our
-expenses, in which case we let the supplier invoice your company directly. This
-can be a great way to make your contribution more concrete and visible ("thank
-you Foocorp Inc. for the lovely t-shirts!"). Keep in mind that in this case we
-might not be able to predict up front the exact amount invoiced, it could for
-instance depend on the amount of sign-ups.
 
-## Budget
+## Financial
+
+There's two ways your company can choose to support us:
+### Direct Sponsorship
+If you've opted for a direct sponsorship tier, then your transaction will be with
+our suppliers directly. In this case, we will give you the exact
+link/message/order that we need, and you make the transaction with that
+company directly. This means normal business billing, with ability to pay by
+credit card, and get an invoice. However, due to the nature of the process,
+we can't predict the exact amount — it would depend on factors such as the
+the number of sign-ups.
 
-This is an example budget based on previous workshops:
+### Support us via donation
```

Comment:
Perhaps we would be willing to take higher amounts from donations? We only have three tiers after all, and with a presumption that Cognitect is probably taking one of those.

I think that mentioning the 250 again would still be warranted, but I'm unsure how then to phrase it.